### DISCUSSION TITLE
Cannot run storybook when using with typescript and vite builder

### DISCUSSION THREAD

**Discussion by mockingjet** [storybookjs/storybook#19196] [Help]
I created this mini repo to demo the issue.
https://github.com/mockingjet/storybook-vite-ts-issue

I use pnpm, vite, vue3, babel, and follow this [tutorial](https://storybook.js.org/docs/vue/builders/vite#typescript). But still get the error:
![image](https://user-images.githubusercontent.com/32839503/190345497-987dad47-ec24-4b64-addb-586db2b3af7a.png)

**Comment by IanVS** [comment]
Typescript config files do not work great with the vite builder in storybook 6.5, unfortunately.  They'll be supported out-of-the-box in 7.0.  For now, you can try installing `ts-node` in your project, or just use a `main.cjs` file for now, until you're comfortable upgrading to storybook 7.0.  If you want to give it a shot now, you can run `npx sb@next upgrade --prerelease`.
